app/javascript/packages/webauthn/enroll-webauthn-device.ts

```diff
@@ -23,6 +23,8 @@ interface EnrollOptions {
   excludeCredentials: PublicKeyCredentialDescriptor[];
 
   authenticatorAttachment?: AuthenticatorAttachment;
+
+  publicKeyCredentialHints?: 'client-device' | 'security-key';
```

Comment:
since this `'client-device' | 'security-key'` union is used in two spots, should we name it & typedef it?

```ts
typedef PublicKeyCredentialType = 'client-device' | 'security-key';
```

Reply:
I like that idea. Could even use an enum and call it the same as in the spec, `PublicKeyCredentialHints`

https://www.w3.org/TR/webauthn-3/#enumdef-publickeycredentialhints
https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Reply:
I have those cast in a Type now. I'm not entirely clear on how to leverage an enum there but can spend a little more time on it.